public/app/plugins/datasource/loki/tracking.ts

```diff
@@ -132,18 +132,34 @@ const shouldNotReportBasedOnRefId = (refId: string): boolean => {
   return false;
 };
 
-export function trackQuery(response: DataQueryResponse, queries: LokiQuery[], app: string): void {
+export function trackQuery(
+  response: DataQueryResponse,
+  request: DataQueryRequest<LokiQuery> & { targets: LokiQuery[] },
+  startTime: Date
+): void {
   // We only want to track usage for these specific apps
+  const { app, targets: queries } = request;
+
   if (app === CoreApp.Dashboard || app === CoreApp.PanelViewer) {
     return;
   }
 
+  let totalBytes = 0;
+  for (const frame of response.data as DataFrame[]) {
```

Comment:
We should probably not do `as DataFrame` and figure out how can we do this without introducing new assertion.

Reply:
Updated in 6bc1e609400ebb50f860b485307b73ea4359faaf.